sashimi-webapp/test/unit/specs/database/generated-data/dateTime.spec.js

```diff
@@ -0,0 +1,17 @@
+import DateTime from 'src/database/generated-data/dateTime';
+
+const dateTime = new DateTime();
+
+describe('dateTime.js', () => {
+  describe('create dateTime string for alasql format', () => {
+    it('should be a string', () => {
+      const currentDateTime = dateTime.getCurrentDateTime();
+      expect(currentDateTime).to.be.a.string;
+    });
+
+    it('should have a date length of 19 characters', () => {
+      const currentDateTime = dateTime.getCurrentDateTime();
+      expect(currentDateTime.length).to.equal(19);
+    });
```

Comment:
I couldn't test date time because my code rely on Date(). But calling Date() might have different seconds and rely on the same dependency, so I just make sure the length of the format is correct. YYYY.MM.DD HH:MM:SS >.<